template/desktops/xfce.sh

```diff
@@ -19,6 +19,39 @@ xfconf-query -c xfce4-session -p /startup/ssh-agent/enabled -n -t bool -s false
 xfconf-query -c xfce4-session -p /startup/gpg-agent/enabled -n -t bool -s false
 echo "TIMING: $(date -Iseconds) - Disabled startup services"
 
+# Turn off power saving measures that turn off the display
+xfconf-query \
```

Comment:
Would adding a comment to the link for the types be good here?

Reply:
I think I'll just link to this PR, since there's a lot of context here, probably more than it makes sense to add in a script, but enough to understand what's happening and fix things if needed.